### PR TITLE
GroupExPro. Rollback migration during manual Sync operation.

### PR DESCRIPTION
**Pull request by ygerasimov**
Currently for GroupExPro we clean up old data (rolling back migration) only during drush command. This PR adds this feature to the form that can be run manually.

Credit to YMCA WNC

### CODE REVIEW

File: modules/custom/openy_gxp/openy_gxp.drush.inc [hunk 1]
```diff
@@ -20,9 +20,6 @@ function openy_gxp_drush_command() {
  * Command 'openy-gxp-sync'.
  */
 function drush_openy_gxp_sync() {
-  // First rollback all the data completely.
-  drush_migrate_tools_migrate_rollback('gxp_offerings_import');
```
Comment (podarok) [inline modules/custom/openy_gxp/openy_gxp.drush.inc:24]: Why is this removed from drush?

Comment (ygerasimov) [reply modules/custom/openy_gxp/openy_gxp.drush.inc:24]: we run it as a part of form submission now. No need to rollback same migration twice

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build1559 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build1559 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build1559 |
| Installation process | http://install.openy.cibox.tools/build1559/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1559/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1559/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1559/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1559/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1559/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1559/blogWCAG2AAhtmlcs.txt
